git_theta/filters.py

```diff
@@ -0,0 +1,146 @@
+"""Clean and Filter functions."""
```

Comment:
Did these change at all? I'm not inspecting them deeply...

Reply:
No, they just got moved so they are more reusable.